README.md

```diff
@@ -1,19 +1,47 @@
 # Google CloudEvents - Go
 
-This repository contains types for CloudEvents issued by Google.
+This library provides classes of common event types used with Google services.
+At this moment the following types are available:
 
-## Prerequisites
+| Package | Struct | Description |
+| ------------- | ------------- | ------------- |
+| auditv1  | AuditLogWrittenEvent | The event is triggered when a new Cloud Audit Log entry is written. |
+| pubsubv1  | MessagePublishedEvent | This event is triggered when a new Cloud Pub/Sub event is published to a topic. |
 
-- Go 1.9 or higher.
-- Go protobuf tool
-  - `go install google.golang.org/protobuf/cmd/protoc-gen-go`
 
-## Generate
+## Installation and Usage
 
-Generate libraries:
+**Note**: This library requires Go 1.11+.
 
-```sh
-./gen.sh
+To install this package, run
+
+``` sh
+go install github.com/googleapis/google-cloudevents-go
 ```
 
-Observe the new files in `src/`.
\ No newline at end of file
+To use an event class, see the snippet below:
+
+``` go
+package main
+
+import (
+	pubsub "github.com/googleapis/google-cloudevents-go/cloud/pubsub/v1"
```

Comment:
Where does the name 'pubsub' come from?

I'd imagine `messagePublishedData` like shown in the CloudEvents source repo, and we wouldn't talk about the service:
https://github.com/googleapis/google-cloudevents